networkx/algorithms/chordal/chordal_alg.py

```diff
@@ -242,6 +244,47 @@ def chordal_graph_treewidth(G):
         max_clique = max(max_clique,len(clique))
     return max_clique - 1
 
+@not_implemented_for('directed','multigraph')
+def chordal_simplicial_vertex(G):
+    """Returns the simplicial vertex of the chordal graph G. 
+    
+    Parameters
+    ----------
+    G : graph
+      A NetworkX graph 
+    
+    Returns
+    -------
+    Simplicial vertex of graphs G is vertex v such that closed neighborhood of v is clique in G.
```

Comment:
This should be

    Returns
    -------
    node
        The simplicial vertex of the graph ``G``.

Maybe "a simplicial vertex"; is it unique?